python-squid/src/test/java/org/sonar/python/parser/python_v3/PythonV3Test.java

```diff
@@ -45,4 +45,14 @@ public void function_declaration(){
     assertThat(p).notMatches("lambda x:Type: x");
   }
 
+  @Test
+  public void function_star_args(){
+    setRootRule(PythonGrammar.TYPEDARGSLIST);
+    assertThat(p).matches("*, foo=bar");
+    assertThat(p).matches("*args, boo");
+    assertThat(p).notMatches("*, boo");
+    assertThat(p).notMatches("*");
+    assertThat(p).notMatches("**");
```

Comment:
@d3matt should match:
`p = 42, **kargs`
`*, p, **kargs`
`p = 1, p2, *, ** kargs`